src/server.tsx

```diff
@@ -153,8 +153,10 @@ app.post(
         const tests = await fetchConfiguredEpicTestsCached();
         const got = findVariant(tests, targeting, targeting.epicViewLog);
 
-        // TODO logging the variant name is not useful, really we want the test + variant names(!)
-        if (got?.test.name !== expectedTest || got?.variant.name !== expectedVariant) {
+        const bothTruthy = expectedTest && got;
+        const notTheSame = got?.test.name !== expectedTest || got?.variant.name !== expectedVariant;
+
+        if (bothTruthy && notTheSame) {
```

Comment:
Is there also a case where one is truthy and the other isn't which we want to capture here? If that's the case could `bothTruthy` be `const eitherTruthy = expectedTest || got`?

Reply:
Ah yes - really we want (not) `bothFalsy`!